### ISSUE TITLE
Can Lerna Update Only the Version in package.json Without Code Changes?

### ISSUE THREAD

**Issue by wopelo** [lerna/lerna#3975]
Is it possible for Lerna to update only the version field in package.json? I have a multi-package project managed by Lerna + yarn workspace with the following structure:
```
/my-monorepo-project
|-- packages
|   |-- package1
|   |-- package2
|   |-- package3
|-- package.json
|-- yarn.lock
|-- lerna.json
```
package1 has dependencies on package2 and package3. I have updated and published an alpha version 1.0.0.alpha of package2 (and consequently package1), and after using it, I found no issues with the alpha version. Now, I want to release the stable version without any code changes. Currently, I manually change the version of package2 to 1.0.0 and use lerna version --no-private. This prompts me to select the next version for both package2 and package1. For package2, I need to select 'custom' and then enter the same version number 1.0.0. Is there a more elegant way to achieve this without manually editing package.json, directly through a Lerna command?
I am using version 7.4.2 of lerna。

**Comment by fahslaj** [comment]
Hi @wopelo , you can pass an explicit version argument to `lerna version`, along with the `--force-publish` argument to force Lerna to version again without code changes:

`lerna version 1.0.0 --no-private --force-publish`

**Comment by wopelo** [comment]
@fahslaj I understand what you mean, but I am using Independent mode. If I use `lerna version 1.0.0 --no-private --force-publish`, wouldn't it update all the sub-packages to version 1.0.0 and force-publish them? If I want to force-publish only specific sub-packages, what should I do?

**Comment by wopelo** [comment]
I've realized that there's an issue with the way I've been using Lerna. After making changes and committing them, I first run 
`lerna version` to modify the version, and then execute `lerna publish from-package`. A more reasonable approach would be to directly execute `lerna publish` after committing the changes, as lerna publish includes the lerna version step.
Returning to the issue at hand, the solution I've found is to manually modify the subpackage version numbers and commit those changes first, then execute `lerna publish from-package`. After a successful publish, run `git tag <package-name>@<version> -m <commit-message>` to tag the commit, and then execute `git push --tags` to push the local tags to the remote repository.
